### ISSUE TITLE
Problem with pretrained model

### ISSUE THREAD

**Issue by Gatsby23** [PRBonn/4DMOS#4]
Dear Professor:
         Thank you for your nice work about identify the moving objects on the ground. However, when I use the pretrained models, it shows that "it has error in loading state_dict for MOSNet". I don't know why and how to solve this problem. Is there some advice?
       
![Screenshot from 2022-07-05 13-33-16](https://user-images.githubusercontent.com/18000675/177256749-8a94143d-1a97-4629-b5f6-f927ddb478ce.png)
 Yours
Qi Wu

**Comment by benemer** [comment]
Dear Qi Wu,

Thank you for your interest in our work. The missing key in the checkpoint state dictionary is the loss weight added in https://github.com/PRBonn/4DMOS/pull/3. We just updated the pre-trained models with this parameter. You can now download and use the models as described in the Readme.

Best
Benedikt

**Comment by Gatsby23** [comment]
> Thank you for your interest in our work. The missing key in the checkpoint state dictionary is the loss weight added in #3. We just updated the pre-trained models with this parameter. You can now download and use the models as described in the Readme.

Sorry for troubling you again: I use the command "python scripts/predict_confidences.py -w model/5_scans.ckpt
" to do the prediction and the 5_scans.ckpt is the model I download from readme. However, it shows the same error again. The torch and cuda I used are all the same with the Dockfile.

**Comment by benemer** [comment]
Did you re-download the model? For me, it does work now. Please verify that you download and use the latest model from our server.

**Comment by Gatsby23** [comment]
> Did you re-download the model? For me, it does work now. Please verify that you download and use the latest model from our server.

OK，I'll redonwload it at once. Thank you very much.

**Comment by benemer** [comment]
Does it work for you now?

**Comment by Gatsby23** [comment]
> Does it work for you now?
![Screenshot from 2022-07-06 20-54-05](https://user-images.githubusercontent.com/18000675/177554838-9ed611dd-1486-47f4-8328-46f2aa7e4752.png)
OK, it's work now

**Comment by Gatsby23** [comment]
> Does it work for you now?

Thank you very much

**Comment by MinChoi0129** [comment]
I had the same error, but I added just one more parameter.
In predict_confidence.py : line 74,

[from]
model.load_state_dict(ckpt["state_dict"])

[to]
model.load_state_dict(ckpt["state_dict"], strict=False)
